sdk/ai/azure-ai-projects/samples/connections/async_samples/sample_connections_async.py

```diff
@@ -114,8 +114,8 @@ async def sample_connections_async():
 
     elif connection.connection_type == ConnectionType.AZURE_AI_SERVICES:
 
-        from azure.ai.inference.aio import ChatCompletionsClient
-        from azure.ai.inference.models import UserMessage
+        from azure.ai.inference.aio import ChatCompletionsClient  #type: ignore
```

Comment:
what are #type ignore comments?

Reply:
We are missing the library stubs and mypy will give an error on it during the import.

Reply:
Removed